src/data/stopwords.py

```diff
@@ -0,0 +1,135 @@
+"""
+Obtained from NLTK's list of english stopwords
+
+Functional requirements: FR8
```

Comment:
is this related to our SRS?

Reply:
Oh sorry, thanks for catching it. No it's not, these are the stopwords from the other group's project. I think they're relevant to include, but not part of our FRs.